pkg/status/endpoints/status.go

```diff
@@ -23,10 +23,11 @@ func PopulateStatus(stats map[string]interface{}) {
 		return
 	}
 
+	deduped := utils.DedupEndpoints(endpoints)
 	endpointsInfos := make(map[string]interface{})
 
 	// obfuscate the api keys
-	for endpoint, keys := range endpoints {
+	for endpoint, keys := range deduped {
 		for i, key := range keys {
```

Comment:
With this and change in `domain_resolver.go` we would only need one dedup function.

```suggestion
	for endpoint, eps := range endpoints {
		for i, key := range utils.DedupEndpoint(eps) {
```